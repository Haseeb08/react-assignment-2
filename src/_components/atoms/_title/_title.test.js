import React from "react";
import { create } from "react-test-renderer";
import { render, screen, getByRole } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Title from "./_title";

describe("Title testing", () => {
  test("Matches the snapshot", () => {
    const title = create(<Title />);
    expect(title.toJSON()).toMatchSnapshot();
  });

  test("Titleis present", () => {
    // Using screen
    render(<Title />);
    const title = screen.getByText("test title")

    expect(title).toBeInTheDocument();
  });
});
