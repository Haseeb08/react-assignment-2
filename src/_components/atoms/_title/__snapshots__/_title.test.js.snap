// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title testing Matches the snapshot 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h2"
>
  test title 
</h2>
`;
